engine/src/test/java/org/camunda/bpm/engine/test/api/authorization/AuthorizationLoggingTest.java

```diff
@@ -71,7 +71,7 @@ public void shouldLogOnDebugLevel() {
     engineRule.getManagementService().toggleTelemetry(true);
 
     // then
-    String message = "ENGINE-03029 Required admin authenticated group or user.";
+    String message = "ENGINE-16002 Exception while closing command context: ENGINE-03110 Required admin authenticated group or user or any of the following permissions:";
```

Comment:
Why do we have that extra "ENGINE-16002 Exception while closing command context:" now? Is that done intentionally by us or where does that now come from?

Reply:
The "ENGINE-16002" code was part of the exception log before. The filter in the logging rule looks for lines containing a given String. We expect it to be in the log message so I added it.

Reply:
That makes sense, thanks for explaining 👍 
I guess we did not include it before to stay independent from general outer behavior that is not relevant for the unit we test here. We could also leave it like that since I believe we test the outer exception message behavior already. But I am not 100% certain.

Reply:
Sure we can also remove it. 👍

Reply:
👍